### PR TITLE
Linkedin post email

### LINKED ISSUE #366: Scrap E-Mail Links from a LinkedIn Post
# Description

Scrap E-Mail Links from a LinkedIn Post. The basic features I am looking at from this Script include: 
- Enter the Link of the Post 
- Scrap the E-Mail of specified subdomains (G-Mail, Yahoo, Outlook, ProtonMail).
- Store them on a CSV File and output the same. 

## Language

- [X] Go
- [X] Javascript
- [X] Python

## Type of issue

_Please delete options that are not relevant._

- [X] Feature

### PR DESCRIPTION
**Pull request by kaustubhgupta**
# Description

This script scraps all the emails from the Linkedin post. These post are mainly done by people sending bulk emails to large audience. This script collects all of them and saves them as CSV file which can be further automated to send emails.

Fixes #366 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Linkedin_post_emails_scrapper/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+`python app.py linkedin_username linkedin_password linkedin_post_link`
+
+Before running the script, make sure you have the required dependicies. Not sure about them? Run the following command before running the actual script:
+
+`pip install -r reqirements.txt`
+
+Also you will require latest chrome driver. Download latest one from [here](https://chromedriver.chromium.org/downloads) and place in the same location where script lies.
+
+# Working Demo
+![](preview.gif)
```
Comment (HarshCasper) [inline Python/Linkedin_post_emails_scrapper/README.md:16]: It should be ```image```

File: Python/Linkedin_post_emails_scrapper/app.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    file_open.close()
+    df = pd.DataFrame(emails)
+    df.to_csv('Data.csv')
+    os.remove('data.txt')
+    driver.close()
+
+if __name__ == "__main__":
+    extract(sys.argv[1], sys.argv[2] , sys.argv[3])
+
+
```
Comment (HarshCasper) [inline Python/Linkedin_post_emails_scrapper/app.py:105]: Remove these lines

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Kindly make the changes

**Comment by kaustubhgupta** [comment]
@HarshCasper made all the required changes!

**Pull request merged**